### PR TITLE
Updated CharacterMetadata for DCC tool needs

### PR DESCRIPTION
**Pull request by mkinzelman-believer**
Added metadata fields to track rigs, animation directories, and meshes.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (rudoi) [inline yarn.lock:1]: i think you may have `yarn`'d in root without realizing it. this file can go away :D

**Review by rudoi — APPROVED** [review]
LGTM, i tagged this for CI so if that passes let's do it!

File: yarn.lock (hunk 1 above)
Comment (mkinzelman-believer) [reply yarn.lock]: Deleted

**Comment by mkinzelman-believer** [comment]
Edited a couple strings in CharacterCard.svelte

**Review by rudoi — APPROVED** [review]

**Pull request merged**